packages/flex-plugin-scripts/src/config/index.ts

```diff
@@ -56,7 +57,11 @@ const getConfiguration = async <T extends ConfigurationType>(
       try {
         return require(getPaths().app.webpackConfigPath)(config, args);
       } catch (exception) {
-        await emitDevServerCrashed(exception);
+        if (script === 'start') {
+          await emitDevServerCrashed(exception);
```

Comment:
What about the dev-server config file?

Reply:
it's in the block right below - only start calls that